lib/qesapdeployment.pm

```diff
@@ -553,7 +553,8 @@ sub qesap_create_aws_credentials {
 sub qesap_create_aws_config {
     my %paths = qesap_get_file_paths();
     my $region = script_output q|awk -F ' ' '/aws_region/ {print $2}' | . $paths{qesap_conf_trgt};
-    $region = get_required_var('PUBLIC_CLOUD_REGION') if ($region =~ /^%.+%$/);
+    $region = get_required_var('PUBLIC_CLOUD_REGION') if ($region =~ /^["']?%.+%["']?$/);
+    $region =~ s/[\"\']//g;
```

Comment:
If you move this line one up, the regexp in the condition in L556 gets easier.

Reply:
but I like it also to take care to remove quotes eventually present in the `PUBLIC_CLOUD_REGION` setting